bot/src/db.rs

```diff
@@ -22,6 +30,13 @@ pub struct ValidatorClassification {
     pub stake_state: ValidatorStakeState,
     pub stake_state_reason: String,
 
+    /// computed score (more granular than ValidatorStakeState)
+    pub epoch_credits: u64, // epoch_credits is the base score
+    pub score_discounts: ScoreDiscounts,
+    pub commission: u8,
+    pub active_stake: u64,
+    pub data_center_concentration: f64,
+
```

Comment:
In order to avoid breaking the existing database entries, the new fields here need to each be wrapped in an `Option<>`.  You might find it easier to moving them all out into a single struct and wrap an `Option` around just that struct